src/App.js

```diff
@@ -1,12 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
-import MainPage from './components/mainPage';
+import "./App.css";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import MainPage from "./components/mainPage";
+import MainStatsEqui from "./components/statEquivalent/mainStatsEqui";
+import ShouldITodd from "./components/shouldITodd";
 
 function App() {
   return (
-    <div className="App">
-        <MainPage />
-    </div>
+    <BrowserRouter>
+      <Routes>
```

Comment:
If you are using router, you need to make sure to re-render the header either on an app level above the routed pages or as a global component within each page.

This is to make sure the top bar actually appears in each page instead of disappearing, because right now all the other pages post route have no header and you literally cannot navigate back.